### PR TITLE
fixes #12

### LINKED ISSUE #12: Pick Up
Hello when you try to pick up the package you can pick it up everywhere Is It to me?

### PR DESCRIPTION
**Pull request by tom-osborne**
simple fix to issue #12
`pickupZonedestroy()` was `pickupZone:destroy()`

### CODE REVIEW

**Comment by Andyauk** [comment]
> simple fix to issue #12 `pickupZonedestroy()` was `pickupZone:destroy()`

nice find

**Comment by milotiro95** [comment]
Sans titre

File: client/main.lua [hunk 1]
```diff
@@ -368,7 +368,7 @@ RegisterNetEvent(
       function()
         ClearPedTasks(PlayerPedId())
         PickupPackage()
-        pickupZone:destroy()
+        pickupZonedestroy()
```
Comment (deandum) [inline client/main.lua:371]: This is not correct. https://github.com/mkafrin/PolyZone/wiki/Destroying-a-zone

**Review by deandum — CHANGES_REQUESTED** [review]
That is not a function. Enable debug for the zones and do the job to see how they work. The zone should be destroyed after you picked up a box.

**Comment by tom-osborne** [comment]
Incorrect fix

**Pull request closed without merging**